examples/preload-aliases.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * If you need to run the Requests 2.x library with WordPress core that still ships v1.x,
+ * use the following to preload the class aliases before WordPress gets a chance to load
+ * the actual implementations of Requests 1.x.
```

Comment:
I wonder if we should make this text a little bit more agnostic ? i.e. less specific to WordPress.
The same preloading could be needed in the context of other CMSes/frameworks.

Reply:
Also, from my previous remark:

> Should probably also include a warning that using a mix of Requests 1.x and 2.x files is a bad idea to start with and to do so at your own risk.